### DISCUSSION TITLE
Localization not working

### DISCUSSION THREAD

**Discussion by PeaceDeveloper** [wger-project/wger#963] [General]
Hi,

I'm using the master's branch of this project. The command 'docker-compose -f extras/docker/compose/docker-compose.yml up' described is working fine. But, after app is loaded in http://localhost:8000/pt/software/features, for example, the portuguese language is not loaded.

What is wrong?

Thank's

**Comment by rolandgeider** [comment]
Hi! Do you mean that none of the strings are translated at all?

**Reply by PeaceDeveloper** [reply]
An example is http://localhost:8000/en/software/features. This page is not translating. While the page http://localhost:8000/en/user/login only the username and password fields are being translated.
